Add Approval API docs for PAG (#21325)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/approval-get.md

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

### For entitlement management

| Permission type                        | Permissions (from least to most privileged) |
|:---------------------------------------|:--------------------------------------------|
| Delegated (work or school account)     | EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | Not supported. |

### For PIM for groups

| Permission type                        | Permissions (from least to most privileged) |
|:---------------------------------------|:--------------------------------------------|
| Delegated (work or school account)     | PrivilegedAssignmentSchedule.Read.AzureADGroup, PrivilegedAssignmentSchedule.ReadWrite.AzureADGroup |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | Not supported. |

## HTTP request

<!-- { "blockType": "ignored" } -->

> ### To get approval example for entitlement management.
> ```http
> GET /identityGovernance/entitlementManagement/accessPackageAssignmentApprovals/{id}
> ```
> 
> ### To get approval example for PIM for groups.
> ```http

Reviewer: ```suggestion
To list the approval steps in entitlement management.
```http
GET /identityGovernance/entitlementManagement/accessPackageAssignmentApprovals/{id}
```

To list the approval steps in PIM for groups:
```http
```
